source/docs/software/commandbased/commands.rst

```diff
@@ -1,67 +1,277 @@
 
-Similarly to subsystems, the recommended method for most users to create a command is to subclass the abstract ``CommandBase`` class (`Java <https://github.wpilib.org/allwpilib/docs/beta/java/edu/wpi/first/wpilibj2/command/CommandBase.html>`__, `C++ <https://github.wpilib.org/allwpilib/docs/beta/cpp/classfrc2_1_1_command_base.html>`__), as seen in the command-based template (`Java <https://github.com/wpilibsuite/allwpilib/blob/main/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/templates/commandbased/commands/ExampleCommand.java>`__, `C++ <https://github.com/wpilibsuite/allwpilib/blob/main/wpilibcExamples/src/main/cpp/templates/commandbased/include/commands/ExampleCommand.h>`__):
+Command Properties
+------------------
+
+In addition to the four lifecycle methods described above, each ``Command`` also has two properties, the value of which is defined by the ``runsWhenDisabled`` and ``getInterruptBehavior`` methods respectively. In most cases, the default value is enough.
+
+getRequirements
+^^^^^^^^^^^^^^^
+
+Each command should declare any subsystems it controls as requirements. This backs the scheduler's resource management mechanism, ensuring that no more than one command requires a given subsystem at the same time. This prevents situations such as two different pieces of code attempting to set the same motor controller to different output values.
+
+Most command factories and constructors provided by the library have a ``requirements`` vararg (Java) / initializer list (C++) parameter.
+
+Declaring requirements is done by overriding the ``getRequirements()`` method in the relevant command class, by calling ``addRequirements()``, or by using the ``requirements`` vararg (Java) / initializer list (C++) parameter at the end of the parameter list of most command constructors and factories in the library.
+
+As a rule, command compositions require the union of the requirements of their component commands.
+
+runsWhenDisabled
+^^^^^^^^^^^^^^^^
+
+The ``runsWhenDisabled()`` method returns a ``boolean``/``bool`` specifying whether the command may run when the robot is disabled. With the default of returning ``false``, the command will be canceled when the robot is disabled and attempts to schedule it will do nothing. Returning ``true`` will allow the command to run and be scheduled when the robot is disabled.
+
+.. important:: Except for Addressable LEDs, hardware outputs are disabled when the robot is disabled, regardless of ``runsWhenDisabled()``!
```

Comment:
This isn't true. Digital Outputs aren't disabled either. I think it would be better to list what is disabled, PWM, CAN motor controllers, solenoids, solenoids.

Reply:
Are you sure about that?

This admonition was taken verbatim from before this PR.

Reply:
It was wrong before this PR then ;-)